src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -17,7 +23,22 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
+    public String getUniqueCharacters(String fileName) throws IOException {
+        int maxLength = 5;
+        TreeSet<Character> treeSet = new TreeSet<>();
+        try {
```

Comment:
try to use `try with resources`

Reply:
Sorry I thought if I put resources into {} it will be already try-with-resources... fixed